web-api/src/storage/elastic.rs

```diff
@@ -284,7 +284,7 @@ impl storage::Document for Storage {
         let body = Some(json!({
             "query": {
                 "ids" : {
-                    "values" : ids
+                    "values" : self.postgres.documents_exist(ids).await?
```

Comment:
:question:  I just realized that now but there might be one problem: For elastic search `filter.id.values` can easily  become way to big to be included into every request for personalized documents :sweat: 

Maybe we need to filter it in postgres after we got the ids, at the risk of returning less then expected documents (Which might be fine given that this should be an rare situation).

Reply:
i would also expect it to be too much work for elastic for larger numbers of ids. i implemented it in this way to uphold the current requirement that we return exactly k documents (if enough exist). can we relax this requirement to ease the load on elastic, @acrrd?